package.json

```diff
@@ -28,6 +28,7 @@
     "editorconfig": "^0.13.2"
   },
   "devDependencies": {
+    "lazy-req": "^1.1.0",
```

Comment:
You are using this in the main code, this should be listed in the  `dependencies`, not the `devDependencies`!